src/main/java/org/commcare/formplayer/application/FormController.java

```diff
@@ -130,19 +127,44 @@ public FormEntryResponseBean changeLocale(@RequestBody ChangeLocaleRequestBean c
     @ConfigureStorageFromSession
     public FormEntryResponseBean answerQuestion(@RequestBody AnswerQuestionRequestBean answerQuestionBean,
                                                 @CookieValue(name=Constants.POSTGRES_DJANGO_SESSION_ID, required=false) String authToken) throws Exception {
-        SerializableFormSession serializableFormSession = formSessionService.getSessionById(answerQuestionBean.getSessionId());
-        FormSession formEntrySession = new FormSession(serializableFormSession, restoreFactory, formSendCalloutHandler, storageFactory);
+
+        SerializableFormSession serializableFormSession = FormplayerSentry.timedBreadcrumb(
+                Constants.TimingCategories.GET_SESSION,
+                () -> formSessionService.getSessionById(answerQuestionBean.getSessionId())
+        );
 
         // add tags for future datadog/sentry requests
         datadog.addRequestScopedTag(Constants.FORM_NAME_TAG, serializableFormSession.getTitle());
         Sentry.setTag(Constants.FORM_NAME_TAG, serializableFormSession.getTitle());
 
-        FormEntryResponseBean responseBean = formEntrySession.answerQuestionToJSON(answerQuestionBean.getAnswer(),
-                answerQuestionBean.getFormIndex());
+        FormSession formEntrySession = FormplayerSentry.timedBreadcrumb(
+                Constants.TimingCategories.INITIALIZE_SESSION,
+                () -> new FormSession(serializableFormSession, restoreFactory, formSendCalloutHandler, storageFactory)
+        );
+
+        FormEntryResponseBean responseBean = FormplayerSentry.timedBreadcrumb(
+                Constants.TimingCategories.PROCESS_ANSWER,
+                () -> formEntrySession.answerQuestionToJSON(
+                        answerQuestionBean.getAnswer(), answerQuestionBean.getFormIndex()
+                )
+        );
+
+        FormplayerSentry.timedBreadcrumb(
+                Constants.TimingCategories.GET_SESSION,
+                () -> formSessionService.getSessionById(answerQuestionBean.getSessionId())
+        );
```

Comment:
Is this duplicate timing necessary for `GET_SESSION` considering the same call is timed a few lines earlier?

Reply:
hmm, no, not sure why that's there.